bigquery/google/cloud/bigquery/magics.py

```diff
@@ -39,6 +39,9 @@
         amount of time for the query to complete will not be cleared after the
         query is finished. By default, this information will be displayed but
         will be cleared after the query is finished.
+    * ``--params <params dictionary>`` (optional, line argument):
+        If present, the argument must be a parsable JSON string. This dictionary
```

Comment:
How about "... must be a JSON string or a reference to dictionary which is serializable to JSON." ?

It's too bad that it has to go to a JSON string, since that means we can't pass in DATETIME or NUMERIC query parameters, but that's probably okay, given that we want the magics to be simple rather than complete.